xc7/tests/soc/litex/mini_ddr/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+add_file_target(FILE mem.init)
+add_file_target(FILE mem_1.init)
+
+add_file_target(FILE arty.sdc)
+add_file_target(FILE arty.json) # IOSTANDARD defs for fasm2bels
```

Comment:
I don't believe the arty.json file is required anymore, delete it?

Reply:
Done